sdk/tables/data-tables/review/data-tables.api.md

```diff
@@ -499,6 +524,7 @@ export class TableServiceClient {
 export type TableServiceClientOptions = PipelineOptions & {
     endpoint?: string;
     version?: string;
+    requestPolicyFactories?: RequestPolicyFactory[] | ((defaultRequestPolicyFactories: RequestPolicyFactory[]) => void | RequestPolicyFactory[]);
```

Comment:
Added this in order to be able to create a client with a pipeline that allows intercepting requests to build the batch

Reply:
Why not have `TableServiceClientOptions` take an `InnerBatchRequest` instead? That way you can have `TableClient` assemble the right policies instead of having them be passed in by the batch itself.

Reply:
I'm trying this out, the only thing I'm not sure about is if we want to expose InnerBatchRequest. What do you think?